logger/logger.go

```diff
@@ -15,7 +18,30 @@ func NewTag(tag string) *slog.Logger {
 
 // New returns a new generic instance of a logger that implements Logger interface
 func New() *slog.Logger {
-	logger := slog.New(slog.NewJSONHandler(os.Stdout, nil))
+	cfg, err := config.New()
+	if err != nil {
+		slog.Error(e.FAIL, "err", err, "description", "Couldn't parse config")
+	}
+
+	var level slog.Level
+	switch cfg.Munch.LogLevel {
+	case "debug":
+		level = slog.LevelDebug
+	case "info":
```

Comment:
Why not remove this case if we are setting the level to info by default anyways?

Reply:
I agree, that's logically redundant. It's there just for clear illustration of the acceptable log levels, while the default handles entries that are not valid. Doesn't hurt, or dirty the code, does it? It can be removed, but it reads better. Keep, or remove?

Reply:
@Yendelevium Aside: I see a different problem here though. If line 21 is true, then line 27's`cfg.Munch.LogLevel` reference will fail because `cfg` object will be nil. It's prudent to always expect a valid config to be returned on calling `config.New()`

Please refer #7, and let's discuss there. But this code need not be changed because line 27 `cfg` will be valid.